src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery ball1 = new Lottery();
+        System.out.println(ball1.getRandomBall().toString());
```

Comment:
Instead of creating each time new instance of Ball, you should create the constant NUMBER_OF_BALLS with value 3. 
Then in the main method, you can create an array of Balls, and use your constant to assign the array length. 
Here by using the for each loop create instances of the Lottery. 
HINT, you can replace the method toString() by overriding this method.